src/components/common/SketchBox.tsx

```diff
@@ -6,9 +6,20 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { Row, Col } from 'reactstrap';
 
+interface SketchBoxProps { 
+  img : string;
```

Comment:
Could you remove all the space before colons in the interface definition please?